master/internal/user/postgres_users.go

```diff
@@ -251,6 +251,15 @@ func deleteAgentUserGroup(
 	return err
 }
 
+// A UserProfileImage row just contains the profile image data. It is probably split into another table to avoid
```

Comment:
can we delete the user profile code

obviously as a separate ticket make a ticket to delete this code?

Reply:
yeah, we can i think.